app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= render 'partials/navigation' %>
+
+<%= form_with(url: user_recipe_recipe_foods_path, model: @recipe_food) do |form| %>
+  <% if @recipe_food.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(@recipe_food.errors.count, "error") %> prohibited this recipe food from being saved:</h2>
+      <ul>
+        <% @recipe_food.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <div>
+    <%= form.label :food_id, style: "display: block" %>
+    <%= form.select :food_id, Food.all.map { |food| [food.name, food.id] } %>
+  </div>
+  <div>
+    <%= form.label :quantity, style: "display: block" %>
+    <%= form.text_field :quantity %> 
+  </div>
+  <div>
+    <%= form.label :unit, style: "display: block" %>
+    <%= form.text_field :unit %> 
+  </div>
+  <div>
+    <%= form.submit %>
+  </div>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you also add some styling on the above form of adding **recipe_foods**, to make your project looks better.